docs/concept/#894-Efficient-orchestration-of-edc-negotiations/#894-Efficient-orchestration-of-edc-negotiations.md

```diff
@@ -0,0 +1,103 @@
+object with a state indicating if the token is valid (i.e. can be used directly to authenticate the dataplane request)
+or expired (i.e. the validityPeriod is expired). After expiration the token can be reused to request a new token. Cache
+eviction is handled independent of token state and will use a configurable TTL to evict cached entries.
+
+![edc-flow-orchestrated.svg](edc-flow-orchestrated.svg)
+
+See [docs/src/uml-diagrams/runtime-view/edc-flow-orchestrated.puml](https://github.com/eclipse-tractusx/item-relationship-service/blob/main/docs/src/uml-diagrams/runtime-view/edc-flow-orchestrated.puml)
+for sources.
+
+### Special Case: Registry Negotiation
+
+Contract offers for the digital twin registry are identified by a type. This changes the process slightly, so it is not
+only one contract offer which will be negotiated, but potentially multiple contract offers.
+
+Since the NegotiationOrchestrator works asynchronously, we can simply start the EDR negotiation process for all the
+received contract offers and will receive a list of Future EDRs.
+
+The process of NegotiationOrchestrator is left out of the chart, since it is the same as previous.
+
+![edc-flow-orchestrated-ddtr.png](edc-flow-orchestrated-ddtr.svg)
+
+See [docs/src/uml-diagrams/runtime-view/edc-flow-orchestrated-ddtr.puml](https://github.com/eclipse-tractusx/item-relationship-service/blob/main/docs/src/uml-diagrams/runtime-view/edc-flow-orchestrated-ddtr.puml)
+for sources.
+
+## Testing Strategy
+
+E2E Tests can be re-used, because the data we receive does not change. The only difference is, that fewer negotiations
+are done.
+
+For unit- and integration tests, the Wiremock tests “SubmodelFacadeWiremockTest” and “IrsWireMockIntegrationTest”
+provide full control over all participating components and can therefore verify every interaction between them.
+
+These tests allow to verify the that Tokens are reused, negotiations for the same Asset are only conducted once and
+simultaneous requests to the DTR are still working.
+The communication to EDC does not change so the already existing *Support Classes can be reused.
+
+New scenarios have to be defined which aim to test the parallel execution of negotiations.
+
+### Test case: start 2 jobs with the same globalAssetId at the same time
+
+Expected result is that both jobs complete, but for the second job, no new negotiations should be started .
+
+Verify that the EDC endpoints for EDC stubs are only interacted with once by using the verification methods of
+WiremockSupport.
+
+Verify that the Cache has been used for the registry and submodel negotiation.
+
+### Test case: start 2 jobs with different globalAssetIds at the same time
+
+Expected result is that both jobs complete and both had their full set of negotiations conducted.
+
+Verify that the Cache has been used for the registry negotiation.
+
+Verify that the submodel negotiations are done for each submodel.****
```

Comment:
Please do not use pictures in the future. Use mermaid instead to prevent deviations between diagram and concept.